### PR TITLE
Ensure relayed returndata is 32 bytes

### PR DESCRIPTION
**Pull request by jordy25519**
Returndata must fit into 32 bytes or reported as invalid.
Restricting the byte length ensures costs are fixed and mitigates DoS potential

### CODE REVIEW

File: .github/workflows/coverage.yml [hunk 1]
```diff
@@ -15,16 +15,19 @@ jobs:
     steps:
```
Comment (jordy25519) [inline .github/workflows/coverage.yml:15]: unrelated fix to coverage workflow

**Review by JasonTulp — APPROVED** [review]
lgtm!
